admin/configure.php

```diff
@@ -105,7 +105,7 @@ function civicrm_backend_config(string $configFile, $adminPath): string {
 ";
 
   $string = trim($string);
-  JFile::write($adminPath . DIRECTORY_SEPARATOR .
+  \Joomla\CMS\Filesystem\File::write($adminPath . DIRECTORY_SEPARATOR .
```

Comment:
Change looks good but breaks J3 compatibility? Need to use version_compare() if we want to maintain this for the time being?